Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -61,7 +61,8 @@ class TopSiteItemCell: UICollectionViewCell {
 
     override init(frame: CGRect) {
         super.init(frame: frame)
-
+        isAccessibilityElement = true
+        accessibilityIdentifier = "TopSite"
```

Comment:
It'd be nice if we could get the tests to work by using the actual labels instead of IDs, but I suppose this is fine for now if that's not a straightforward change.